src/main/resources/templates/login_page.html

```diff
@@ -1,22 +1,29 @@
 <!DOCTYPE html>
 <html xmlns:th="http://www.thymeleaf.org" lang="en">
-<head>
-    <head th:replace="fragments/header :: header"></head>
+<head th:replace="fragments/header :: header"></head>
```

Comment:
Removed unnecessary nested <head> tags to simplify the HTML structure, enhancing readability and maintainability. This cleanup aligns with clean code principles discussed in ["Java Clean Code"](https://www.baeldung.com/java-clean-code), which recommend avoiding redundant elements that clutter code without adding value.